cmd/walletinfo.go

```diff
@@ -74,7 +75,7 @@ var signCmd = &cobra.Command{
 		} else {
 			data = encryption.Hash(data)
 		}
-		sign, err := client.Sign(data)
+		sign, err := sys.Sign(data, client.GetClient().SignatureScheme, client.GetClientSysKeys())
```

Comment:
We don't need this change if the gosdk defined the client.Sign function as I suggested